src/index.js

```diff
@@ -68,6 +68,23 @@ function runAsync (command, options) {
   })
 }
 
+function runAsyncSpawn (command, args, options) {
+  return new Promise((resolve, reject) => {
+    const spawnOptions = Object.assign({shell: true}, options)
+    const asyncProcess = childProcess.spawn(command, args, spawnOptions)
```

Comment:
Nice, that'll mean this won't be a breaking change. I'll address your comments this weekend.